src/Action/GetShortObjectDescriptionAction.php

```diff
@@ -75,7 +75,9 @@ public function __invoke(Request $request): Response
                 'id' => $admin->id($object),
                 'label' => $admin->toString($object),
             ]]);
-        } elseif ('html' === $request->get('_format')) {
+        }
+
+        if ('html' === $request->get('_format')) {
```

Comment:
Can you explain this change a bit? why is it needed?

Reply:
nothing to do with PHPStan. That was my choice.

The code that looks like this
```
public function bar(string $arg): Baz
{
    if($arg === 'a') {
        return 'something';
    } elseif ($arg === 'b') {
        return 'something1';
    } elseif ($arg === 'c') {
        return 'something2';
    } elseif ($arg === 'd') {
        return 'something3';
    }
}
```
Looks more tricky and complex that this one
public function bar(string $arg): Baz
{
    if ($arg === 'a') {
        return 'something';
    }

    if ($arg === 'b') {
        return 'something1';
    }

    if ($arg === 'c') {
        return 'something2';
    }

    if($arg === 'd') {
        return 'something3';
    }
}
```

-----

I can revert it if you want

Reply:
Removing a useless else if seems ok to me